pkg/fleet/installer/config/config_windows.go

```diff
@@ -0,0 +1,140 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package config
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/pkg/fleet/installer/telemetry"
+)
+
+const (
+	deploymentIDFile = ".deployment-id"
+)
+
+// GetState returns the state of the directories.
+func (d *Directories) GetState() (State, error) {
+	stablePath := filepath.Join(d.StablePath, deploymentIDFile)
+	experimentPath := filepath.Join(d.ExperimentPath, deploymentIDFile)
+	stableDeploymentID, err := os.ReadFile(stablePath)
+	if err != nil && !os.IsNotExist(err) {
+		return State{}, fmt.Errorf("error reading stable deployment ID file: %w", err)
+	}
+	experimentDeploymentID, err := os.ReadFile(experimentPath)
+	if err != nil && !os.IsNotExist(err) {
+		return State{}, fmt.Errorf("error reading experiment deployment ID file: %w", err)
+	}
+	return State{
+		StableDeploymentID:     string(stableDeploymentID),
+		ExperimentDeploymentID: string(experimentDeploymentID),
+	}, nil
+}
+
+// WriteExperiment writes the experiment to the directories.
+func (d *Directories) WriteExperiment(ctx context.Context, operations Operations) error {
+	state, err := d.GetState()
+	if err != nil {
+		return fmt.Errorf("error getting state: %w", err)
+	}
+	if state.ExperimentDeploymentID != "" {
+		return fmt.Errorf("there is already an experiment in progress")
+	}
+	err = os.RemoveAll(d.ExperimentPath)
+	if err != nil {
+		return fmt.Errorf("error removing experiment directory: %w", err)
+	}
+	err = os.MkdirAll(d.ExperimentPath, 0700)
```

Comment:
is this expected to be `C:\ProgramData\Datadog-exp` ? if so we'll have to adjust how it's created/removed and has its permissions applied, similar to [here](https://github.com/DataDog/datadog-agent/blob/81d499cfb4fcd8d997187db298da9f7db90e4b57/pkg/fleet/installer/paths/installer_paths_windows.go#L212-L215). We (Windows Products) can do this in a follow up PR.

Reply:
right, I think that's what is making the tests unhappy lol

Reply:
sgtm, I'll move it under `Installer`